devel/pulp/devel/test_runner.py

```diff
@@ -12,6 +12,27 @@
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pulp.server.webservices.settings'
 
 
+# From: http://stackoverflow.com/questions/377017/test-if-executable-exists-in-python/377028#377028
```

Comment:
Can this link be moved into a docstring for the function? Also can you add some new lines linking to the [license of the content](http://stackoverflow.com/help/licensing) and [that license itself](https://creativecommons.org/licenses/by-sa/3.0/). This snippet will need to retain that license.

Another option is to make a new implementation that would be licensed like the rest of Pulp as gplv2.